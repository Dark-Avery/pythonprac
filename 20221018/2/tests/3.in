:sin x y sin(x+y)
sin 1 -2
:decorate s w "--<<{},{}>>--".format(s,w)
decorate "ЖЖЖ" "УУУ"
sin 0 1
quit "все будет хорошо!! {}, {}"