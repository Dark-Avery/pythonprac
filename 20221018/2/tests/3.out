-0.8414709848078965
--<<ЖЖЖ,УУУ>>--
0.8414709848078965
все будет хорошо!! 3, 6
